src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import cn from "classnames";
+import styles from "./Navbar.module.scss";
+
+export const Navbar = () => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const closeNavbar = () => setIsOpen(false);
+
+  return (
+    <nav role="navigation" aria-label="menu" className={styles.navbar}>
+      <div
+        className={cn(styles.menuList, {
+          [`${styles.menuList}-active`]: isOpen,
+        })}
+      >
+        <p>Menu</p>
+        <ul>
+          <li onClick={closeNavbar} className={styles.menuListItem}>
+            <Link to="/menu">Twój jadłospis</Link>
+          </li>
+          <li onClick={closeNavbar} className={styles.menuListItem}>
+            <Link to="/recipes">Przepisy</Link>
+          </li>
+          <li onClick={closeNavbar} className={styles.menuListItem}>
+            <Link to="/shopping-list">Lista zakupów</Link>
+          </li>
+          <li onClick={closeNavbar} className={styles.menuListItem}>
+            <Link to="/recipes">Twój profil</Link>
+          </li>
+        </ul>
+      </div>
+      <button
+        className={styles.hamburgerButton}
+        aria-controls="navigation"
+        aria-expanded={isOpen}
+        onClick={() => setIsOpen((prevIsOpen) => !prevIsOpen)}
```

Comment:
Toggle navbar is sometimes little tricky because you never know what state will be after toggle() function :D 
CloseNav() will always be closed and openNav() will always be opened 😆

Reply:
yeah, but here all we wanna do is toggling 😄 we have the same button, that's changing the state to opposite when clicked

Reply:
Zgadzam się :D Tak tylko mówie o tym jakie miałem przykre doświadczenia z tym XD